docs/config.mdx

```diff
@@ -81,13 +81,6 @@ Invitation link shown:
 | `CALLSTACKS`             | Displays the stack trace for DiscordSRV debug calls                                                                                           |
 :::
 
-### [`DebugLevel`](https://config.discordsrv.com/config/DebugLevel) {#DebugLevel}
-- `0` - Disables debug messages
-- `1` - Enables debug messages
-- `2` - Enables debug messages w/ Stack Trace
-
-Note: DebugLevel is now deprecated
-
```

Comment:
needs to be reverted since it's already being handled in this PR: https://github.com/DiscordSRV/Documentation/pull/68